notary-server-Dockerfile

```diff
@@ -19,4 +19,4 @@ RUN go install \
     ${NOTARYPKG}/cmd/notary-server
 
 ENTRYPOINT [ "notary-server" ]
-CMD [ "-config", "cmd/notary-server/config.json" ]
+CMD [ "-config", "cmd/notary-server/config.json", "-debug"]
```

Comment:
Shouldn't have `-debug` set in the Dockerfile (incase somebody deploys it to production unintentionally)